unit_tests/storage/test_bucket.py

```diff
@@ -560,10 +562,12 @@ def __init__(self):
         blob = _Blob()
         with _Monkey(ACL, save=save):
             new_blob = source.copy_blob(blob, dest, NEW_NAME, client=client,
-                                        preserve_acl=True)
+                                        preserve_acl=False)
             self.assertIs(new_blob.bucket, dest)
             self.assertEqual(new_blob.name, NEW_NAME)
             self.assertIsInstance(new_blob.acl, ObjectACL)
+            self.assertFalse(new_blob.acl.loaded)
+            self.assertEqual(new_blob.acl.entities, {})
```

Comment:
Because it hasn't been loaded, having it be empty here is not really asserting anything useful.  In fact, we need to be asserting that the second API call is actually made (to save the emptied ACL).